common/djangoapps/student/tests/test_email.py

```diff
@@ -406,10 +406,9 @@ def test_email_success(self):
             subject='Request to change édX account e-mail',
             body_fragments=[
                 'We received a request to change the e-mail associated with',
-                'your édX account from {old_email} to {new_email}.'.format(
-                    old_email=old_email,
-                    new_email=new_email,
-                ),
+                'your édX account from',
+                old_email,
+                new_email,
```

Comment:
why have you updated these formatted strings?

Reply:
because the inline styling was applied to emails 

Here is how [it used in the template](https://github.com/openedx/edx-platform/pull/33421/files#diff-b8bb580e7199b9bf3f468adea436aea700130a220c6ce6be8dcb7d06cfd9a876R17) and actual [changes for the email variables](https://github.com/openedx/edx-platform/pull/33421/files#diff-b8bb580e7199b9bf3f468adea436aea700130a220c6ce6be8dcb7d06cfd9a876R6)